### PR TITLE
Add generate-stackbrew-library.sh

### PR DESCRIPTION
**Pull request by J0WI**
This script outputs the format used by the [docker library](https://github.com/docker-library/official-images/tree/master/library).

<details>
<summary>Example output</summary>

```
# This file is generated via https://github.com/phpmyadmin/docker/blob//generate-stackbrew-library.sh
Maintainers: Isaac Bennetch <bennetch@gmail.com>
             Michal Čiha <michal@cihar.com>
GitRepo: https://github.com/phpmyadmin/docker.git

Tags: 4.8.5-apache, 4.8-apache, 4-apache, 4.8.5, 4.8, 4
Architectures: amd64, arm32v7, arm64v8, i386, ppc64le
GitCommit: 78e3cb7aba41b233281f81d8b264d95505c43c96
Directory: apache

Tags: 4.8.5-fpm, 4.8-fpm, 4-fpm
Architectures: amd64, arm32v7, arm64v8, i386, ppc64le
GitCommit: 78e3cb7aba41b233281f81d8b264d95505c43c96
Directory: fpm

Tags: 4.8.5-fpm-alpine, 4.8-fpm-alpine, 4-fpm-alpine
Architectures: amd64, arm32v7, arm64v8, i386, ppc64le
GitCommit: 78e3cb7aba41b233281f81d8b264d95505c43c96
Directory: fpm-alpine
```
</details>

### CODE REVIEW

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+	if [ "$variant" = "apache" ]; then
+		variantAliases+=( "${versionAliases[@]}" )
+	fi
+
+	variantParent="$(awk 'toupper($1) == "FROM" { print $2 }' "$variant/Dockerfile")"
+
+	# depends on docker library
+	#variantArches="${parentRepoToArches[$variantParent]}"
+	variantArches="amd64 arm32v7 arm64v8 i386 ppc64le"
```
[the remark this answers is not in the document]
Comment (J0WI) [reply generate-stackbrew-library.sh]: Nope: https://github.com/docker-library/official-images#multiple-architectures

**Comment by J0WI** [comment]
:ping_pong: @ibennetch

**Comment by williamdes** [comment]
The tags will appear after merging ?

**Comment by J0WI** [comment]
:robot:

**Comment by williamdes** [comment]
Thanks for your contribution on adding `generate-stackbrew-library.sh` !
Merged :)

**Pull request merged**